packages/composite-checkout/src/components/billing-fields.js

```diff
@@ -0,0 +1,557 @@
+					/>
+				) : (
+					<Field
+						id={ location + '-postal-code' }
+						type="Text"
+						label={ localize( 'Postal code' ) }
+						error={ eval( zipCode + 'Error' ) }
+						errorMessage={ localize( 'This is a required field' ) }
+						value={ eval( postalCode ) }
+						onChange={ updatePostalCode }
+					/>
+				) }
+
+				<Field
+					id={ location + '-country' }
+					type="Text"
+					label={ localize( 'Country' ) }
+					error={ eval( country ) + 'Error' }
+					errorMessage={ localize( 'This is a required field' ) }
+					value={ eval( country ) }
+					onChange={ updateCountry }
+				/>
+			</FieldRow>
+		</React.Fragment>
+	);
+}
+
+TaxFields.propTypes = {
+	paymentData: PropTypes.object,
+	setPaymentData: PropTypes.func,
+	location: PropTypes.string,
+};
+
+function isZipOrPostal() {
+	//TODO: Add location detection to return "zip" or "postal"
+	return 'postal';
+}
+
+function DomainFields( { paymentData, setPaymentData } ) {
+	const localize = useLocalize();
+
+	return (
+		<DomainContactFields>
+			<DomainContactFieldsTitle>
+				{ localize( 'Enter your domain registration contact information' ) }
+			</DomainContactFieldsTitle>
+			<DomainContactFieldsDescription>
+				{ localize( `Domain owners have to share contact information in a public database of all domains. With
+				our Privacy Protection, we publish our own information and privately forward any
+				communication to you.` ) }
+			</DomainContactFieldsDescription>
+
+			<AddressFields
+				paymentData={ paymentData }
+				setPaymentData={ setPaymentData }
+				location={ 'domains' }
+			/>
+			<TaxFields
+				paymentData={ paymentData }
+				setPaymentData={ setPaymentData }
+				location={ 'domains' }
+			/>
+			<PhoneNumberField
+				paymentData={ paymentData }
+				setPaymentData={ setPaymentData }
+				location={ 'domains' }
+			/>
+		</DomainContactFields>
+	);
+}
+
+DomainFields.propTypes = {
+	paymentData: PropTypes.object,
+	setPaymentData: PropTypes.func,
+};
+
+const DomainContactFields = styled.div`
+	margin: 16px 0 24px;
+`;
+
+const DomainContactFieldsTitle = styled.h2`
+	font-size: 16px;
+	margin: 0 0 4px;
+	font-weight: 600;
+	color: ${props => props.theme.colors.borderColorDark};
+	padding-top: 24px;
+	border-top: 1px solid ${props => props.theme.colors.borderColorLight};
+`;
+
+const DomainContactFieldsDescription = styled.p`
+	font-size: 14px;
+	color: ${props => props.theme.colors.textColor};
+	margin: 0 0 16px;
+`;
+
+function BillingFormSummary( { paymentData } ) {
+	const localize = useLocalize();
+
+	//Check if paymentData is empty
+	if ( Object.entries( paymentData ).length === 0 ) {
+		return null;
+	}
+
+	return (
+		<GridRow gap="4%" columnWidths="48% 48%">
+			<div>
+				<BillingSummaryParagraphs>
+					{ paymentData.billingName } <br />
+					{ paymentData.billingAddress } <br />
+					{ paymentData.billingCity }, { paymentData.billingState } <br />
+					{ paymentData.billingZipCode || paymentData.billingPostalCode },{' '}
```

Comment:
I think we could use a template string here to avoid the awkward `{' '}`. Something like this:

```js
{ `${paymentData.billingZipCode || paymentData.billingPostalCode}, ${paymentData.billingCountry}` }
```

Reply:
yeah, that looks much cleaner. Thanks for the suggestions.